CreeDictionary/API/search/__init__.py

```diff
@@ -1,15 +1,7 @@
-from sortedcontainers import SortedSet
+from .runner import search
 
-from .lookup import (
-    WordformSearchWithAffixes,
-    WordformSearchWithExactMatch,
-)
-from .types import SearchResult
 
-
-def search_with_affixes(
-    query: str, include_auto_definitions=False
-) -> SortedSet[SearchResult]:
+def search_with_affixes(query: str, include_auto_definitions=False):
```

Comment:
Consider putting the return type back into this function definition

Reply:
I prefer not to explicitly specify types that mypy can infer. I have updated the `.serialize()` method that is called here to be annotated with a type instead. Now mypy correctly infers the return type as `list[SerializedPresentationResult]`.